.eslintrc.js

```diff
@@ -190,6 +190,46 @@ module.exports = {
         "@typescript-eslint/prefer-readonly-parameter-types": "off",
       },
     },
+    {
+      files: "*.mdx",
+      extends: ["plugin:mdx/recommended"],
+      rules: {
+        "react/jsx-filename-extension": ["error", { extensions: [".mdx"] }],
+        "mdx/no-unescaped-entities": "OFF",
```

Comment:
Hi, I’m the author of `eslint-mdx`, so glad that you’re using it.

Just to confirm, is there any problem with this rule or you’d just want to ignore it?

Reply:
Hi! Thanks so much for having a look at our configuration. When the rule is turned off, I get a lot of warnings about apostrophes and the like being able to be escaped. I don't want to escape things in markdown.

These cases are markdown inside components ([example](https://github.com/kiwicom/orbit/pull/2716/files#diff-97ba94fb83d933c8f2c77510e1b46bcb1d49f4376e86aa275c195a25301a9b2eR24). I turned it off to get only relevant warnings.

Reply:
That's interesting, I thought `mdx` as `jsx` more than `md`, but users like you may think oppositely. So I'm considering disable it by default in the recommended config.

Reply:
I don't want to speak for others, but in my case I definitely think of it as `md` with extra features. But I'm a writer first and a programmer second.